src/cpp/conf/rsession-dev.conf

```diff
@@ -27,6 +27,10 @@ r-resources-path=${CMAKE_CURRENT_SOURCE_DIR}/session/resources
 r-session-library=${CMAKE_CURRENT_BINARY_DIR}/r/R/packages/library
 r-session-package-archives=${RSTUDIO_DEPENDENCIES_DIR}/common
 
+# override home and doc dirs
+r-home-dir-override=${LIBR_HOME}
+r-doc-dir-override=${LIBR_DOC_DIR}
+
```

Comment:
The only issue with doing this is that it makes it hard for us to test features of rserver that dynamically change the version of R being run (e.g. the user profile feature in rstudio server pro). There is another conf/-dev pair called "rdesktop-dev" that I believe does what you want (and contains the -override flags already).